lib/readline.js

```diff
@@ -390,7 +390,10 @@ Interface.prototype._tabComplete = function() {
         var width = completions.reduce(function(a, b) {
           return a.length > b.length ? a : b;
         }).length + 2;  // 2 space padding
-        var maxColumns = Math.floor(self.columns / width) || 1;
+        var maxColumns = Math.floor(self.columns / width);
+        if (!maxColumns || maxColumns === Infinity) {
+          maxColumns = 1;
+        }
```

Comment:
There's a getter defined for columns: https://github.com/nodejs/node-v0.x-archive/blob/master/lib/readline.js#L165-L170